docs/introduction.rst

```diff
@@ -10,9 +10,11 @@ PHP management for their own use.
 Installation
 ~~~~~~~~~~~~
 
-This role requires at least Ansible ``v2.0.0``. To install it, run::
+This role requires at least Ansible ``v2.0.0``. To install it, run:
 
-    ansible-galaxy install debops.php
+.. code-block:: yaml
+
+   user@host:~$ ansible-galaxy install debops.php
```

Comment:
:wink: Also maybe remove the "user@host:~$ " ? The the command could be highlighted as Bash or Shell command (maybe in the future there will be better support here.). For now, I think it is just semantically better to avoid the "user@host:~$ ".